src/test/java/uk/gov/hmcts/reform/cosapi/edgecase/event/privatelaw/CreateCaseEventTest.java

```diff
@@ -0,0 +1,106 @@
+package uk.gov.hmcts.reform.cosapi.edgecase.event.privatelaw;
+
+import com.google.common.collect.ImmutableSetMultimap;
+import com.google.common.collect.SetMultimap;
+import org.junit.Before;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.ccd.sdk.ConfigBuilderImpl;
+import uk.gov.hmcts.ccd.sdk.api.Event;
+import uk.gov.hmcts.ccd.sdk.api.Permission;
+import uk.gov.hmcts.reform.cosapi.common.AddSystemUpdateRole;
+import uk.gov.hmcts.reform.cosapi.edgecase.model.CaseData;
+import uk.gov.hmcts.reform.cosapi.edgecase.model.State;
+import uk.gov.hmcts.reform.cosapi.edgecase.model.UserRole;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.anyList;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.ccd.sdk.api.Permission.C;
+import static uk.gov.hmcts.reform.cosapi.constants.CommonConstants.CREATE_CASE_EVENT_ID;
+import static uk.gov.hmcts.ccd.sdk.api.Permission.R;
+import static uk.gov.hmcts.reform.cosapi.edgecase.model.UserRole.CASE_WORKER;
+import static uk.gov.hmcts.ccd.sdk.api.Permission.U;
+import static uk.gov.hmcts.reform.cosapi.edgecase.model.UserRole.CITIZEN;
+import static uk.gov.hmcts.reform.cosapi.util.ConfigTestUtil.createCaseDataConfigBuilder;
+import static uk.gov.hmcts.reform.cosapi.util.ConfigTestUtil.getEventsFrom;
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest
+@ActiveProfiles("test")
+@SuppressWarnings("PMD")
```

Comment:
is this tag required to suppress warnings?

Reply:
This suppression has been removed.